packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -75,6 +77,7 @@ export default class VscodeHatRenderer {
   ) {
     extensionContext.subscriptions.push(this);
 
+    this.decoder = new TextDecoder("utf-8");
```

Comment:
nit: do this on the same line as the declaration above?

Reply:
Done.